Turn/src/Player.cpp

```diff
@@ -122,6 +122,53 @@ int Player::Attack(){
 	}
 }
 
+void Player::UseItem() {
+	// Use item from inventory
+	int choice = 0;
+
+	while (true) {
+		ClearScreen();
+
+		// Displays the inventory.
+		DisplayInventory();
+
+		// Gives player a list of moves to choose from.
+		cout << "Choose which item use:" << endl
+			<< "1) Use Potion" << endl
+			<< "2) Use Whetstone" << endl
+			<< "0) Quit" << endl << endl;
+
+		choice = input();
+
+		// Evaluates player's choice.
+		switch (choice) {
+		case 0:
+			return;
+		case 1:
+			// Player drinks a potion.
+			// Does not execute if there are no potions in the inventory.
+			if (potions > 0) {
+				UsePotion();
+				Sleep(SLEEP_MS);
+			}
```

Comment:
If the player has 0 potions there should be a message displayed here, followed by `Sleep(SLEEP_MS)`